src/System.IO.FileSystem/src/Resources/Strings.resx

```diff
@@ -131,7 +131,10 @@
   </data>
   <data name="Arg_InvalidHandle" xml:space="preserve">
     <value>Invalid handle.</value>
-  </data>  
+  </data>
+  <data name="Arg_MustBeDriveLetterOrRootDir" xml:space="preserve">
+    <value>Drive name must be a root directory ('C:\\') or a drive letter ('C').</value>
```

Comment:
I think escaping backslashes is not needed here, also I'd change it to `(i.e. 'C:\)'`

Reply:
thank's!